crates/dbsp/src/circuit/dbsp_handle.rs

```diff
@@ -404,12 +436,14 @@ enum Command {
     EnableProfiler,
     DumpProfile,
     RetrieveProfile,
+    Commit(usize),
```

Comment:
Why is a commit number a `usize`? I would have guessed `u64` (though they're really the same type on the systems we care about).

Reply:
fixed